### PR TITLE
Add code coverage badge

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

File: source/geod24/LocalRest.d [hunk 1]
```diff
@@ -1344,3 +1344,39 @@ unittest
...
+        override int ping () { return 42; }
+
+        override void check ()
+        {
+            auto node = new RemoteAPI!API(node_tid, 420.msecs);
+
+            // Requests are dropped, so it times out
+            assert(node.ping() == 42);
+            node.ctrl.sleep(10.msecs, true);
+            auto exc = collectException!Exception(node.ping());
```
Comment (AndrejMitrovic) [inline source/geod24/LocalRest.d]: `assertThrown!Exception(node.ping())`?

**Comment by Geod24** [comment]
Weird it does not show the expected line as Hit...

**Comment by AndrejMitrovic** [comment]
Looks good.

**Comment by Geod24** [comment]
I'll break this further down

**Pull request merged**